src/sparse_merkle_tree/in_memory_smt.rs

```diff
@@ -229,6 +230,29 @@ impl JsFunctionContext<'_> {
         Ok(proof)
     }
 
+    fn get_removing_keys_parameters(&mut self) -> NeonResult<RemovingParameters> {
+        let proof = self.get_proof(0)?;
+
+        let removing_keys = self
+            .context
+            .argument::<JsArray>(1)?
+            .to_vec(&mut self.context)?;
+        let mut parsed_removing_keys = NestedVec::new();
+        for key in removing_keys.iter() {
+            let key = key
+                .downcast_or_throw::<JsTypedArray<u8>, _>(&mut self.context)?
+                .as_slice(&self.context)
+                .to_vec();
+            parsed_removing_keys.push(key);
+        }
+        let callback = self
```

Comment:
If there is a blank line above to separate extraction of different arguments (0 and 1), maybe we should add a blank line also here? Or we just don't use blank lines in this function?

Reply:
Done.